Makefile

```diff
@@ -662,6 +662,13 @@ test-e2e: ## Run "docker-build" and "docker-push" rules then run e2e tests.
 	$(MAKE) docker-build docker-push \
 	test-e2e-run
 
+.PHONY: test-e2e-existing
+test-e2e-existing:
+	$(MAKE) set-manifest-image MANIFEST_IMG=$(CONTROLLER_IMG)-$(ARCH) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./config/default/manager_image_patch.yaml" MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG)
+	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./config/default/manager_pull_policy.yaml" PULL_POLICY=IfNotPresent
+	MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG) \
```

Comment:
does MANAGER_IMAGE need to be set both here and above on line 667?

Reply:
Actually, I think we only need to set it on `make test-e2e-run` for an envsubst.